packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -350,6 +522,22 @@ function handler(app: Application) {
       await handler.cleanDatastoreTable(watchedPRs, app, context);
       return;
     }
+
+    //because we're searching for the PRs, and not getting the installation ID, we have to use
+    //the bot's installation ID to call the API. So, we need to make sure it matches the repo owner
+    if (context.payload.cron_org) {
+      logger.info('Entering job to pick up any hanging PRs');
+      const owner = context.payload.organization.login;
+      const repo = context.payload.repository.name;
+
+      if (context.payload.cron_org !== owner) {
+        logger.info(`skipping run for ${context.payload.cron_org}`);
+        return;
+      }
+      await handler.pickUpPR(owner, repo, context.github);
+      return;
+    }
+
     const start = Date.now();
     await handler.checkPRMergeability(watchedPRs, app, context);
```

Comment:
Is it worth trying to do these both async and awaiting them at the end? (it's going to complete all of `pickUpPR` before trying to do the merges.)

Reply:
Oh, I'm expecting this to run on a separate cron job - we won't try merging on the same run as we try picking up the PRs.